app/services/markdown_processor/parser.rb

```diff
@@ -127,6 +128,22 @@ def escape_liquid_tags_in_codeblock(content)
       end
     end
 
+    def add_proper_code_tags(content)
+      # return content if there is not a <code> tag or there are triple back ticks
+      return content if /<code>/.match?(content) == false || /```/.match?(content)
```

Comment:
You can use a negation instead of explicitly comparing to `false`:

```suggestion
      return content if !/<code>/.match?(content) || /```/.match?(content)
```

If you find that hard to read you can also swap the check from `Regexp#match?` to `String#match?`, i.e. `!content.match?(/code/)`. I actually think this form is generally more readable so maybe you could update it throughout the method.

Reply:
Followed your advice but Rubocop gave me an offense, so decided to use "unless" instead of "if !"